index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <header>
+        JS Documentation
+      </header>
+      <!--This area of the code shows the navigation of the page-->
```

Comment:
Please, it would be best to remove comments from your codebase. If not, your code may be seen as unclean.
Good job too so far.